test/api/handlers_test.go

```diff
@@ -472,6 +477,67 @@ func testUserLimits(t *testing.T, at *test.AccountsTester) {
 	if tl.TierName != database.UserLimits[database.TierAnonymous].TierName {
 		t.Fatalf("Expected tier name '%s', got '%s'", database.UserLimits[database.TierAnonymous].TierName, tl.TierName)
 	}
+	if tl.DownloadBandwidth != database.UserLimits[database.TierAnonymous].DownloadBandwidth {
+		t.Fatalf("Expected download bandwidth '%d', got '%d'", database.UserLimits[database.TierAnonymous].DownloadBandwidth, tl.DownloadBandwidth)
+	}
+
+	// Create a new user which we'll use to test the quota limits. We can't use
+	// the existing one because their status is already cached.
+	u2, c, err := test.CreateUserAndLogin(at, t.Name()+"2")
+	if err != nil {
+		t.Fatal("Failed to create a user and log in:", err)
+	}
+	defer func() {
+		if err = u2.Delete(at.Ctx); err != nil {
+			t.Error(errors.AddContext(err, "failed to delete user in defer"))
+		}
+	}()
+	at.Cookie = c
+	defer func() { at.Cookie = nil }()
```

Comment:
I'd probably guarantee that we are resetting the cookies on the tester any anything else that could be persisted between tests in the parent test that is calling this subtest.

Reply:
I would *really* prefer to do this in #140 in order to avoid the merge conflicts.